README.md

```diff
@@ -132,8 +132,10 @@ $ letsdane -r https://easyhandshake.com:8053 -skip-dnssec -skip-icann
 
 ### DANE-EE Sites
  
-* FreeBSD: https://freebsd.org
-* Tor Project: https://torproject.org
+* FreeBSD: https://freebsd.org (Free Unix-like operating system)
+* Tor Project: https://torproject.org (Anonymizing web browser)
+* danectl: https://raf.org/danectl (DNSSEC DANE implentation manager)

```

Comment:
Would it be more appropriate to add danectl and huque's tools under a new section like `DANE Tools`?